.yarn/patches/@metamask-assets-controllers-patch-55c0923b82.patch

```diff
@@ -0,0 +1,2243 @@
+diff --git a/dist/TokenRatesController.d.ts b/dist/TokenRatesController.d.ts
```

Comment:
Does this patch to `@metamask/assets-controller` have corresponding PR(s) in https://github.com/metamask/core? Is there a reason to still prefer patching in all the changes here, rather than upgrading from version 26?

Reply:
Related:
- #24333
- #24285
- #24261

Reply:
hey @legobeat ,
thnx for your comment, you can find [here](https://github.com/MetaMask/core/pull/4206) the core PR, i also added it to the PR description.
we use a patch to ship this asap feature without having to bump the assets-controller to the latest version.
once a new release of the assets controller has been made ( core PR merged ), we'll remove the patch and bump the version.